charts/kuberpult/Earthfile

```
@@ -0,0 +1,104 @@
+VERSION 0.7
+
+ARG --required --global VERSION
```

It is really annoying if I have to always pass the version when running earthly. You can set a default value for the argument as what is defined in [Makefile.variables](https://github.com/freiheit-com/kuberpult/blob/main/Makefile.variables#L17)
```suggestion
LOCALLY
ARG --global VERSION=$(git describe --always --long --tags || echo 0.0.1)
```